base/BUILD

```diff
@@ -18,10 +19,9 @@ load(
     "intellij_unit_test_suite",
 )
 
-java_library(
+kt_jvm_library(
```

Comment:
Can we keep bazelrc as a separate library?

Reply:
why?

Reply:
Base is quite big already, just to speed up compilation time. Also, compilation time of base would be even slower if we started conversion to kotlin